src/ResponseSender/SendResponseEvent.php

```diff
@@ -93,9 +93,9 @@ public function setHeadersSent()
     {
         $response = $this->getResponse();
         $headersSent = $this->getParam('headersSent', []);
-        $headersSent[spl_object_hash($response)] = true;
+        $headersSent[$responseObjectHash = spl_object_hash($response)] = true;
```

Comment:
Same, extract variable assignment to a separate line

Reply:
@Xerkus done ;)